src/postgres/src/test/regress/expected/yb_partition_aggregate.out

```diff
@@ -0,0 +1,474 @@
+--
```

Comment:
Here's the naming convention that we follow for regress tests:
1. For tests that are copied from PG as is, we name the file `yb_pg_xx` (in this case `yb_pg_partition_aggregate.out`
2. For new test cases being added specifically for YB, we use a separate file `yb_xx`.

In this case, it seems like you're using postgres tests as is. So, `yb_pg_partition_aggregate` is more appropriate. (Same comment applies to other test files in this diff).

Reply:
Done!